private/src/styles/blocks/postlist/_categories-editor.scss

```
@@ -7,7 +7,7 @@
   align-items: flex-start;
   box-sizing: border-box !important;
   margin: 0 !important;
-  padding: 0 20px !important;
+  padding: 20px !important;
```

we should probably use the theme.json stuff, now that it's available.
```suggestion
  padding: var(--wp--preset--spacing--single) !important;
```